examples/demos/boot_from_san_and_migration/README.md

```diff
@@ -0,0 +1,28 @@
+# Demo:
```

Comment:
Perhaps remove this line? It is already inside the demos folder, kind of expected to be a demo.

Reply:
Ok. Done